zerver/lib/api_test_helpers.py

```diff
@@ -229,8 +229,10 @@ def get_stream_id(client):
 def delete_stream(client, stream_id):
     # type: (Client, int) -> None
 
+    # {code_example|start}
+    # Delete the stream by a given ID
```

Comment:
This description is poor English; "Delete the stream with ID stream_id" would be valid.